src/pages/individualGallery/IndividualGallery.jsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { useParams } from 'react-router-dom';
+import { capitalize, transform } from 'lodash-es';
+
+import { useTheme } from '@material-ui/core/styles';
+
+import useDocumentTitle from '../../hooks/useDocumentTitle';
+import useIndividual from '../../models/individual/useIndividual';
+import { deriveIndividualName } from '../../utils/nameUtils';
+import errorTypes from '../../constants/errorTypes';
+import CustomAlert from '../../components/Alert';
+import Button from '../../components/Button';
+import Link from '../../components/Link';
+import LoadingScreen from '../../components/LoadingScreen';
+import MainColumn from '../../components/MainColumn';
+import SadScreen from '../../components/SadScreen';
+import Text from '../../components/Text';
+import GalleryItem from './components/GalleryItem';
+
+function transformToAssets(individualData) {
+  if (!individualData?.encounters) return [];
+
+  const dataByAsset = transform(
+    individualData.encounters,
+    (memo, encounter) => {
+      const sharedEncounterData = {
+        owner: {
+          guid: encounter?.owner?.guid,
+          fullName: encounter?.owner?.full_name,
+        },
+        time: {
+          time: encounter?.time,
+          timeSpecificity: encounter?.timeSpecificity,
+        },
+        location: {
+          label: encounter?.locationId_value,
+          decimalLatitude: encounter?.decimalLatitude,
+          decimalLongitude: encounter?.decimalLongitude,
+        },
+      };
+
+      // Each asset needs information from the annotation that it belongs to.
+      // Every encounter annotation will be used, but multiple annotations may have the same asset.
+      if (encounter?.annotations) {
+        encounter.annotations.forEach(annotation => {
+          const {
+            guid,
+            bounds,
+            ia_class: iAClass,
+          } = annotation || {};
+          const annotationData = { guid, bounds, iAClass };
+          const assetGuid = annotation?.asset_guid;
+
+          if (assetGuid) {
+            if (!memo[assetGuid]) {
```

Comment:
`if (assetGuid && !memo[assetGuid])`

Reply:
Do you prefer
```
  if (assetGuid && !memo[assetGuid]) // some code
  
  if (assetGuid && memo[assetGuid]) {
    // some other code that requires an asset guid and assumes memo[assetGuid]) already exists
  }
```
instead of
```
  if (assetGuid) {
    if (!memo[assetGuid]) {
       // some code
    }
    // some other code that requires an asset guid and assumes memo[assetGuid]) already exists
  }
```

Reply:
eh, nah, this is fine